apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.html

```diff
@@ -1,6 +1,35 @@
 <div class="header">
-    <div class="logo">
-        
+  <div class="logo">
+    <img src="assets/igus-logo.svg" alt="" />
+  </div>
+  <div class="menu-section">
+    <div class="menu-options">
+        <div class="menu-option"><p><i class="fa fa-envelope"></i></p><p>Contact</p></div>
```

Comment:
Use angular material and not fontawesome

Reply:
Okay